.changeset/lucky-papayas-reflect.md

```diff
@@ -0,0 +1,6 @@
+---
+'@storefront-ui/vue': minor
+'@storefront-ui/react': minor
+---
+
+add ProductSlider block
```

Comment:
Block is not part of our packages that we release on npm, so it should not be mentioned in changelog

Reply:
Yea, was wondering about that - how will we raise the version number when there are no changes to the package? 😄

Reply:
What do you mean, you did changes in component from package in scrollable.vue so you should make patch change only for vue package and write `Prop typing change in SfScrollable`

Reply:
Yup, I did it already yesterday: https://github.com/vuestorefront/storefront-ui/pull/2819/files#diff-d78d5a6ea45c0e27ea4974822b530c55251c860f32d1f9cb9435a4d09d575cc8R2-R5

But what I mean - we need to release the vue/react packages with `2.4` next week - how do we do that?

Reply:
I've remove this changelog. But the issue of "how and when to set version at `2.4`" still persists 😄

Reply:
With what I test today changesets bump version to 2.4 so at this point we need to only go with current new flow and at the end run yarn publish by hand and thats it

Reply:
yea, but we need to make sure that we will always have a minor bump somewhere in our tasks 😄